pkgs/by-name/gi/git-gamble/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  git,
+  installShellFiles,
+  nix-update-script,
+}:
+
+let
+  version = "2.9.0";
+
+  src = fetchFromGitLab {
+    owner = "pinage404";
+    repo = "git-gamble";
+    rev = "version/${version}";
+    hash = "sha256-hMP5mBKXcO+Ws04G3OxdYuB5JoaSjlYtlkerRQ6+bXw=";
+  };
+in
+rustPlatform.buildRustPackage {
+  pname = "git-gamble";
+  inherit version src;
+
+  cargoHash = "sha256-vrzcNdLY2PkyZ1eLwOiONRHVAolbTDxytEgi09WkDZQ=";
+
+  nativeCheckInputs = [ git ];
+  preCheck = ''
+    patchShebangs tests/editor/fake_editor.sh
+  '';
+
+  nativeBuildInputs = [ installShellFiles ];
+  postInstall = ''
+    export PATH="$PATH:$out/bin/"
+
+    sh ./script/generate_completion.sh target/release/shell_completions/
+    installShellCompletion --bash target/release/shell_completions/git-gamble.bash
+    installShellCompletion --fish target/release/shell_completions/git-gamble.fish
+    installShellCompletion --zsh target/release/shell_completions/_git-gamble
```

I think this can be simplified as follows:
```suggestion
    installShellCompletion --cmd git-gamble \
      --bash target/release/shell_completions/git-gamble.bash \
      --fish target/release/shell_completions/git-gamble.fish \
      --zsh target/release/shell_completions/_git-gamble
```